worker.go

```diff
@@ -242,14 +244,26 @@ func doWork(certsProvider certs.CertificatesProvider, certsPerALB int, certTTL t
 		return fmt.Errorf("doWork failed to get certificates: %v", err)
 	}
 
+	cwAlarms, err := getCloudWatchAlarms(kubeAdapter, cwAlarmConfigMapLocation)
+	if err != nil {
+		// We will just log potential errors while loading the alarm
+		// configuration. A non-existent config map or missing service account
+		// permissions should not cause the ingress controller to stop working.
+		// Instead, we just treat the configuration as non-existent and move
+		// on.
+		log.Warnf("ignoring cloudwatch alarm configuration due to error: %v", err)
```

Comment:
I don't agree with this. We should be explicit about the errors and not hide them as warnings IMO. I would rather we only try to lookup a configmap if the use specifies it via the flag, and if not we don't attempt the get config call at all.

Reply:
I get your point. However, if you check the `getCloudWatchAlarms` func you can see that it only performs config lookup if the config map location is non-nil (which is only the case when the user provided it explicitly). How should potential errors be handled here? We could just log the error with level Error and move on (as in the current implementaion) or return it. The latter case would cause the controller to stop working.

Reply:
Got it! Will change that tomorrow.

Reply:
Pushed a change just now.